client/www/pages/docs/auth/magic-codes.md

```diff
@@ -177,27 +177,6 @@ on the backend during permission checks.
 
 On the client, `useAuth` will set `isLoading` to `false` and populate `user` -- huzzah!
```

Comment:
We should update the cases in the docs where we use `db.useAuth`, in exchange for `db.useUser` and db.SignedIn et al?